packages/scene-composer/stories/Matterport.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import SceneComposer, { argTypes, args } from './components/scene-composer';
+import { testScenes } from '../tests/testData';
+import { COMPOSER_FEATURES } from '../src/interfaces/feature';
+
+<Meta title="Tests/Matterport" component={SceneComposer} argTypes={argTypes} />
+
+export const defaultArgs = {
+  source: 'local',
+  scene: 'scene1',
+  theme: 'dark',
+  mode: 'Viewing',
+  density: 'comfortable',
+  matterportModelId: 'cZowU1FxWEQ',
```

Comment:
This is a public model we added to our own account. The application key isn't stored in the source code, but when applied to the story it loads great.